assets/subworkflows/espresso/processing_find_minima.yml

```diff
@@ -0,0 +1,23 @@
+application:
```

Comment:
Same as the above - we should add a comment and declare what this subworkflow expects